km/km_vcpu_run.c

```diff
@@ -435,8 +436,15 @@ static int hypercall(km_vcpu_t* vcpu, int* hc_ret)
    if (km_collect_hc_stats != 0) {
       clock_gettime(CLOCK_MONOTONIC, &start);
    }
-   km_hc_ret_t ret = km_hcalls_table[hc](vcpu, hc, ga_kma);
-   *hc_ret = ga_kma->hc_ret;
+   km_hc_ret_t ret = HC_CONTINUE;
+   if (km_hcalls_table[hc] != NULL) {
+      ret = km_hcalls_table[hc](vcpu, hc, ga_kma);
+      *hc_ret = ga_kma->hc_ret;
```

Comment:
Move the assignment to *hc_ret out of the if then else since both sides do it and both sides have ga-kma->hc_ret already set.

Reply:
OK